City-of-Helsinki/paatokset-search
# Pp 236 search form

To test: 
- Run `npm i && npm start`
- Free text search should execute only when submit button is pressed
- Date select should work with bot predefined ranges as well as with datepicker'
- Category select should work and these selections should be rendered to the bottom of the page
- Pagination creates the right amount of pages (1 in mobile, 5 in desktop) and works

Known issues:
- Other than free text search, selections execute query on select
- No tranlations for static strings as of yet
